DcWebxdcWidget/DcWebxdcWidget.swift

```diff
@@ -0,0 +1,166 @@
+import WidgetKit
+import SwiftUI
+import DcCore
+
+struct Provider: TimelineProvider {
+    func placeholder(in context: Context) -> UsedWebxdcEntry {
+        let limit: Int
+        switch context.family {
+        case .systemSmall:
+            limit = 4
+        case .systemMedium:
+            limit = 7
+        default:
+            limit = 7
```

Comment:
wondering, btw, where the "7" comes from - why not "8"?

Reply:
It comes from my confused brain. `8` is totally right, of course, let me fix that! 🫠